src/igvfd/audit/file_set.py

```diff
@@ -65,22 +65,22 @@ def audit_files_associated_with_incorrect_fileset(value, system):
     if 'files' in value:
         for file in value['files']:
             if file.startswith('/sequence-files/'):
-                sequence_file_object = system.get('request').embed(file, '@@object?skip_calculated=true')
+                sequence_file_object = system.get('request').embed(file)
 
                 # Audit the file set with sequence files without the associated seqspec also in the file set.
                 if sequence_file_object.get('seqspec'):
-                    if sequence_file_object.get('seqspec') not in value['files']:
-                        seqspec_path = sequence_file_object['seqspec']
-                        detail = (
-                            f'File set {audit_link(path_to_text(value["@id"]), value["@id"])} has sequence file '
-                            f'{audit_link(path_to_text(file), file)} which links to seqspec '
-                            f'{audit_link(path_to_text(seqspec_path), seqspec_path)} which does not link to this file set.'
-                        )
-                        yield AuditFailure('missing related files', detail, level='ERROR')
+                    for s in sequence_file_object.get('seqspec'):
```

Comment:
please rename s to something more meaningful (like spec_file or something like that)

Reply:
or config_file